### DISCUSSION TITLE
A lot of things just don't work for me...

### DISCUSSION THREAD

**Discussion by Kasuna89** [tailwindlabs/tailwindcss#12642] [Help] [answered]
... Tailwind goes more and more on my nerves.... but i really wanna try it out... For every step i try to do i need half an hour figuring out how I can solve a problem...
First i made the tailwind.config.js like in the tutorial and did the path correctly, it didn't work at all... then i installed the full tailwind.config.js, did the same thing, it worked... now i wanted to try this:

`<h2 class="text-gray-700 text-xl">Recipes</h2>`

The darker gray color is shown, but not the bigger text... also all of the other colors are not working for me and i don't find anything in the web.... Do i really need to add them seperately? 

I'm really clueless at the moment and really need help or maybe a better tutorial?! Every tutorial i made didn't work properly without problems.. not even the one on the mainpage of tailwindcss.... >_> I'm also available in Discord when someone can help over video or something like that ^^'

Best regards
Kasu

OS: Windows 11
Editor: VSC

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class name would resolve to a Tailwind class that Tailwind would generate a rule for.

Otherwise, consider providing a project that reproduces the unexpected behavior.

**Reply by Kasuna89** [reply]
Thx for the answer, but i don't think i can really follow you. Can you maybe put it for noobs? English isn't my main language so it is a bit difficult for me to understand what you mean. 

Edit:I think i got it and i think none of the things you posted apply here.

Maybe you can take a look at this files. Pls pay attention to the note in the Readme-File.

Project: https://github.com/Kasuna89/yt_tutorial

**Reply by wongjn** [reply]
I am getting a 404 for that link – perhaps you have not made the repository public?

**Reply by Kasuna89** [reply]
Now?

**Reply by wongjn** [reply]
Thank you! I have given it a go, and all seems to work:

- `text-xl` class works, and applies the CSS as expected:
  ![image](https://github.com/tailwindlabs/tailwindcss/assets/11310624/aa096e01-acc5-45c0-84b6-72ccbf2978d4)
- Classes with color values other than `gray-700` work. Here is `red-700` working:
  ![image](https://github.com/tailwindlabs/tailwindcss/assets/11310624/281155bb-c317-4411-849a-d1e6f392226d)

Consider checking you:
- Refresh the page after making changes in `public/index.html`.
- Have cleared your browser cache or have the browser cache disabled while working.
- Have the `watch` npm script running while making changes in `public/index.html`.

**Reply by Kasuna89** [reply]
I use the npm live-server. 
I cleared the browser cache and it is still not working...
What is the watch npm script?

It still looks like this for me... I can't figure it out what i do wrong :/ Everytime i try to install Tailwindcss something doesn't work or i need to add a postscript.config.js, sometimes not... I just don't get it @.@

![screenshot_tailwindcss_failure](https://github.com/tailwindlabs/tailwindcss/assets/42219906/00e68674-72d6-4fac-a6c3-e602a2b4af26)

**Reply by Kasuna89** [reply]
Tailwindcss is applied but it doesn't do any other changes rather the "normal" stuff i believe...

**Reply by wongjn** [reply]
> What is the watch npm script?

[The `watch` script you have defined in your `package.json`](https://github.com/Kasuna89/yt_tutorial/blob/0c612e51fbdbd9df138584c34ff34850abf966ed/package.json#L7).

**Reply by wongjn** [reply]
> Tailwindcss is applied but it doesn't do any other changes rather the "normal" stuff i believe...

Consider checking you:
- Refresh the page after making changes in `public/index.html`.
- Have the `watch` npm script running while making changes in `public/index.html`.

**Reply by Kasuna89** [reply]
Ok i tried ```npx postcss ./src/css/tailwind.css -o ./public/output.css --watch ``` and the following error message came "Input Error: You must pass a valid list of files to parse"

When i tried ```npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch``` the following came "Specified input file ./src/input.css does not exist." ..... but it worked for you so i'm mega confused.

**Reply by wongjn** [reply, marked as answer]
Are you running those verbatim in the terminal? Why not run the command `npm run watch` instead? Isn't being able to run the command `npm run watch` succinctly the reason you have the `watch` script defined in `scripts` in the first place?

Seems like you may not have run either command from the project root, hence the input errors being returned.

**Reply by Kasuna89** [reply]
OMG NOW it is working <3

I didn't even know i had to use that command :o I didn't work with npm for a long time, so still in the learning process ^^'

Why is that command not provided in the tailwind css install guide?
